syntax = "proto3";

service Booking {
    rpc GetUserBookings(UserId) returns (BookingData) {}
    rpc AddToUserBookings(BookingPayload) returns (BookingData) {}
}

message BookingData {
    string userid = 1;
    repeated Date dates = 2;
}

message Date {
    string date = 1;
    repeated string movies = 2;
}


message UserId {
    string userid = 1;
}

message BookingPayload {
    string userid = 1;
    string date = 2;
    string movieid = 3;
}